obspy/io/arclink/tests/test_inventory_xml.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Test suite for the sc3ml reader.
+
+Modified after obspy.io.stationXML
+    > obspy.obspy.io.stationxml.core.py
+
+:author:
+    Mathijs Koymans (koymans@knmi.nl), 11.2015 - [Jollyfant@GitHub]
+
+:license:
+    GNU Lesser General Public License, Version 3
+    (http://www.gnu.org/copyleft/lesser.html)
+"""
+
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from future.builtins import *  # NOQA
+
+import inspect
+import os
+import unittest
+
+from obspy.core.inventory import read_inventory
+from obspy.io.arclink.inventory import validate_arclink_xml
+
+
+class SC3MLTestCase(unittest.TestCase):
```

Comment:
Just a nit-pick.. but this should probably read `InventoryXMLTestCase` (also change at the bottom, if you change it)